### ISSUE TITLE
bugwarrior pulls in merged pull requests but leaves them pending

### ISSUE THREAD

**Issue by bowlofeggs** [ralphbean/bugwarrior#377]
I turned on ```github.involved_issues = True``` today and noticed that it left merged pull requests as "pending" tasks. For example, this pull request was closed long ago:

https://github.com/psss/did/pull/91

However, it was pulled in as pending:

```
$ task 461
No command specified - assuming 'information'.

Name             Value                                                                
ID               461
Description      91: Support did yesterday.                                           
                   2016-09-02 12:37:30 @psss - Thanks for the patch, Randy!           
Status           Pending
Project          did                                                                  
Entered          2016-09-02 12:37:30 (15min)
Last modified    2016-09-02 12:37:30 (15min)                                          
Virtual tags     ANNOTATED ORPHAN PENDING READY UDA UNBLOCKED LATEST PROJECT PRIORITY
UUID             725afd7f-e34e-45c9-ae42-ddf766053f28                                 
Urgency           7.8
Priority         H                                                                    
[githubbody      fixes #79]
[githubcreatedon 1456877280]                                                          
[githubnumber    91]
[githubrepo      psss/did]                                                            
[githubtitle     Support did yesterday.]
[githubtype      pull_request]                                                        
[githubupdatedat 1459773748]
[githuburl       https://github.com/psss/did/pull/91]                                 

    project             1 *    1 =      1
    annotations       0.8 *    1 =    0.8
    UDA priority.H      1 *    6 =      6
                                   ------
                                      7.8

Date                Modification                                               
2016-09-02 12:37:30 Annotation of '@psss - Thanks for the patch, Randy!' added.
```

I assume this might have happened with issues as well, but I did not find any examples in my tasks (there are a ton of them now, and also it failed to finish pulling due to #376).

**Comment by gdetrez** [comment]
If any service fails, bugwarrior will abort and will not touch the tasks(see issue #279), which might explain why the PR is not marked as done.

**Comment by beav** [comment]
I hit this issue as well with 1.4.0, I pulled in a number of PRs but the merged PRs were still marked as 'Pending'.

**Comment by beav** [comment]
@gdetrez The merged PRs still appear when I tested with that.

FWIW here is my config entry for github:

```
[github]
service = github
github.username = beav
github.login = beav
github.token = <token>
add_tags = github
github.import_labels_as_tags = True
github.label_template = GH_{{label}}
github.involved_issues = True
github.filter_pull_requests = True
github.include_user_issues = False
```

**Comment by beav** [comment]
Altering the query statement in this method seemed to help

```python
def get_involved_issues(username, auth):
    """ username should be a string
    auth should be a tuple of username and password.
    """

    #tmpl = "https://api.github.com/search/issues?q=involves%3A{username}&per_page=100"
    tmpl = "https://api.github.com/search/issues?q=involves%3A{username}%20state%3Aopen&per_page=100"
    url = tmpl.format(username=username)
    return _getter(url, auth, subkey='items')

```

**Comment by gdetrez** [comment]
@beav: it looks like that might indeed be the problem. Could you open a pull request with your changes?

**Comment by beav** [comment]
https://github.com/ralphbean/bugwarrior/pull/389

**Comment by ryneeverett** [comment]
Closed via #389.
